src/reason-parser/reason_lexer.mll

```diff
@@ -481,10 +481,15 @@ rule token = parse
   | "#=<" {
     (* Allow parsing of foo#=<bar /> *)
     set_lexeme_length lexbuf 2;
-    SHARPOP("#=")
+    SHARPOP_WITH_EQUAL
   }
   | "#" operator_chars+
-      { SHARPOP(lexeme_operator lexbuf) }
+      { let l = lexeme_operator lexbuf in
+        if l = "#=" then
```

Comment:
I wonder if we should add `| "#=" { SHARPEQUAL }` for clarity in the lexer. No hard opinions here.